src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -29,8 +30,12 @@ def _consume_errors(lib):
         err_lib = lib.ERR_GET_LIB(code)
         err_func = lib.ERR_GET_FUNC(code)
         err_reason = lib.ERR_GET_REASON(code)
+        err_text_reason = ffi.string(lib.ERR_error_string(code, ffi.NULL))
```

Comment:
mmm, so, this will occur on _all_ errors, I wonder if the speed difference of calling this all the time is measurable. Are there any success cases (aka common cases) where we get an OpenSSL error.